Validation/Configuration/python/globalValidation_cff.py

```diff
@@ -120,11 +120,15 @@
 )
 globalValidationTrackingOnly = cms.Sequence()
 
-def _modifyGlobalValidationForPhase2( theProcess ):
-    theProcess.load('Validation.Configuration.gemSimValid_cff')
-    theProcess.load('Validation.Configuration.me0SimValid_cff')
-    theProcess.globalValidation += theProcess.gemSimValid
-    theProcess.globalValidation += theProcess.me0SimValid
+from Validation.Configuration.gemSimValid_cff import *
+from Validation.Configuration.me0SimValid_cff import *
+
+_run3_globalValidation = globalValidation.copy()
+_run3_globalValidation += gemSimValid
+
+_phase2_globalValidation = _run3_globalValidation.copy()
+_phase2_globalValidation += me0SimValid
 
 from Configuration.StandardSequences.Eras import eras
-modifyConfigurationStandardSequencesGlobalValidationForPhase2_ = eras.phase2_muon.makeProcessModifier( _modifyGlobalValidationForPhase2 )
+eras.run3_GEM.toReplaceWith( globalValidation, _run3_globalValidation )
+eras.phase2_muon.toReplaceWith( globalValidation, _phase2_globalValidation )
```

Comment:
@kpedro88 good catch! I had forgotten about the other syntax for `toModify`. Thinking about it I see

1. `toModify` can not be used to remove the sequence from the process while a `toReplaceWith` could

1. when writing a function passed to `toModify` you have to be careful to never call `=` since that does not change the object that was passed in and instead just changes what object the variable _points_ to. E.g.
```
def _func(seq):
   seq = seq+ foo

eras.bar.toModify( fii, _func)
```
would not actually change `fii` since the `_func` just changed the assignment of `seq` and didn't modify the object referred to by `fii`.